languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -10084,6 +10084,378 @@ though, and has since been slightly modified)-->
+              <suggestion>\1, \2</suggestion>
+              <example correction="or, il">Il ne réagit pas au temps <marker>or il</marker> est pressé.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <marker>
+                      <token regexp="yes">[^,);:(«»-]+<exception postag="V.*|A.*" postag_regexp="yes"/><exception regexp="yes">peine|mal</exception></token>
+                      <token>à</token>
+                      <token>savoir</token>
+                  </marker>
+                  <token postag="D.*" postag_regexp="yes"/>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1, \2 \3</suggestion>
+              <example correction="famille, à savoir">Il viendra avec sa <marker>famille à savoir</marker> sa femme et ses deux enfants.</example>
+              <example><marker>Reste à savoir</marker> le prix.</example>
+          </rule>
+          <rule>
+              <pattern>
+                  <marker>
+                      <token regexp="yes">[^,);:(«»-]+</token>
+                      <token>c'</token>
+                      <token>est</token>
+                      <token>-</token>
+                      <token>à</token>
+                      <token>-</token>
+                      <token>dire</token>
+                  </marker>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1, \2\3\4\5\6\7</suggestion>
+              <example correction="temps, c'est-à-dire">Il ne réagit pas au <marker>temps c'est-à-dire</marker> aux saisons qui passent.</example>
+          </rule>
+          <rule>
+              <antipattern>
+                  <token>être</token>
+                  <token>soit</token>
+                  <token max="1" postag="M fin" spacebefore="no">.</token>
+              </antipattern>
+              <pattern>
+                  <marker>
+                      <token postag="V.*|R dem m s|A.*|J.*" postag_regexp="yes"><exception regexp="yes">ne|ce|ça|cela|alors|bien</exception></token>
+                      <token>soit</token>
+                  </marker>
+                  <token postag="M fin" spacebefore="no">.<exception scope="next">.</exception></token>
+              </pattern>
+              <message>Une virgule semble être requise dans ce contexte.</message>
+              <suggestion>\1, \2</suggestion>
+              <example correction="arrive, soit">Il <marker>arrive soit</marker>.</example>
+              <example correction="plus, soit">Il ne le voit <marker>plus soit</marker>.</example>
+              <example>Il ne le voit <marker>plus soit</marker>...</example>
+              <example>Il le faut, <marker>alors soit</marker>.</example>
+              <example>Il n'a de conflit avec qui que <marker>cela soit</marker>.</example>
+          </rule>
+          <rule>
```

Comment:
I see that you've started to take care of the whole maths context, but you'll see a lot of FPs from mathematical texts in there:

> Soit un entier.